hpy/tools/autogen/conf.py

```diff
@@ -139,7 +139,7 @@
     'HPyBool_FromBool': 'PyBool_FromLong',
     'HPy_Compile_s': None,
     'HPy_EvalCode': 'PyEval_EvalCode',
-    'HPyContextVar_New': None,
+    'HPyContextVar_New': 'PyContextVar_New',
```

Comment:
Ah, I didn't properly parse the previous diff. My suggestion was to just remove lines 142 and 144.